scripts/qa_crowbarsetup.sh

```diff
@@ -3742,11 +3742,11 @@ function oncontroller_testsetup()
     nova flavor-delete m1.smaller || :
     nova flavor-create m1.smaller 11 512 8 1
     nova delete testvm  || :
-    nova keypair-add --pub_key /root/.ssh/id_rsa.pub testkey
+    nova keypair-add --pub-key /root/.ssh/id_rsa.pub testkey
     nova secgroup-add-rule default icmp -1 -1 0.0.0.0/0
     nova secgroup-add-rule default tcp 1 65535 0.0.0.0/0
     nova secgroup-add-rule default udp 1 65535 0.0.0.0/0
-    timeout 10m nova boot --poll --image $image_name --flavor $flavor --key_name testkey testvm | tee boot.out
+    timeout 10m nova boot --poll --image $image_name --flavor $flavor --nic net-name=fixed --key-name testkey testvm | tee boot.out
```

Comment:
this changes the meaning.. how is the addition of --nic net-name=fixed related ?